examples/pip_parse/BUILD

```diff
@@ -71,18 +63,22 @@ compile_pip_requirements(
     requirements_txt = "requirements_lock.txt",
 )
 
+filegroup(
+    name = "yamllint_pyz",
```

Comment:
This is not being used?

Reply:
Ah yeah, this was no longer needed after `@rules_python//python/runfiles` was setup.